docs/cli/server.md

```diff
@@ -208,11 +208,11 @@ The ```bor server``` command runs the Bor client.
 
 - ```netrestrict```: Restricts network communication to the given IP networks (CIDR masks)
 
-- ```nodekey```:  P2P node key file
+- ```maxpendpeers```: Maximum number of pending connection attempts (defaults used if set to 0)
 
-- ```nodekeyhex```: P2P node key as hex
+- ```nat```: NAT port mapping mechanism (any|none|upnp|pmp|extip:<IP>)
```

Comment:
I think this is some error, because `maxpendpeers` and `nat` are already defined above, and not sure why `nodekey` and `nodekeyhex` is removed.